### PR TITLE
Add Travis

### LINKED ISSUE #3: Add Travis
Here is an example from the other PHP: https://github.com/expectedbehavior/php-docraptor/pull/23

### PR DESCRIPTION
**Pull request by janxious**
Fixes #3 

Figured we could have some automated testing!

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  - 5.5
+  - 5.6
+  - '7'
+  - hhvm
+script:
+  - script/test
+before_install:
+  - composer self-update --no-interaction
+before_script:
+  - composer install --no-interaction
```
Comment (jqr) [inline .travis.yml:13]: This is mostly in `script/test` already, just without the `--no-interaction` bit.

**Comment by jqr** [comment]
I'm meh on badges, but if you're into 'em, go for it. :+1:

File: .travis.yml (hunk 1 above)
Comment (janxious) [inline .travis.yml:13]: Yeah, I noticed that. Kinda feel like script/setup would be nice to have, but one thing at a time. Until this particular bit bites us, I think it's okay.

**Comment by janxious** [comment]
I'm positive on badges just because if there are some and they are green, it indicates some minimum level of effort on our behalf to rpresent we are doing testing and making good choices. I suspect we will at most have 2 badges on any of these: test runner and some kind of code inspection thing.

**Pull request merged**